packages/plugin-template/src/index.ts

```diff
@@ -1,29 +1,36 @@
-import jsPsych from "jspsych";
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
 
-const plugin = <any>{};
-
-plugin.info = {
+const info = <const>{
   name: "PLUGIN-NAME",
   parameters: {
     parameter_name: {
-      type: jsPsych.plugins.parameterType.INT, // BOOL, STRING, INT, FLOAT, FUNCTION, KEY, SELECT, HTML_STRING, IMAGE, AUDIO, VIDEO, OBJECT, COMPLEX
+      type: ParameterType.INT, // BOOL, STRING, INT, FLOAT, FUNCTION, KEY, KEYS, SELECT, HTML_STRING, IMAGE, AUDIO, VIDEO, OBJECT, COMPLEX
       default: undefined,
     },
     parameter_name2: {
-      type: jsPsych.plugins.parameterType.IMAGE,
+      type: ParameterType.IMAGE,
       default: undefined,
     },
   },
 };
 
-plugin.trial = function (display_element, trial) {
-  // data saving
-  var trial_data = {
-    parameter_name: "parameter value",
-  };
+type Info = typeof info;
 
-  // end trial
-  jsPsych.finishTrial(trial_data);
-};
+/* Plugin description and author */
```

Comment:
Yes, cool! I added `@author <name>`, `@file <plugin description>` and `@see {@link <URL> docs}` to the JsDoc comments for the all plugin classes, including the template. Does that look ok?

Reply:
Very neat! :tada: The `@file` tag is meant to be used at the top of a file, but we annotate a class here, so [`@description`](https://jsdoc.app/tags-description.html) would be the better alternative. Since it's at the beginning, we can omit it as well (and what about setting the lowercase plugin name bold? :smile: ):
```ts
/**
 * **html-button-response**
 * 
 * jsPsych plugin for displaying a stimulus and getting a button response
 * 
 * @author Josh de Leeuw
 * @see {@link https://www.jspsych.org/plugins/jspsych-html-button-response/ html-button-response plugin documentation on jspsych.org}
 */
```

Reply:
Yes, love it! Done for all plugins 👍